pipelines/zap/pipeline.yml

```diff
@@ -1,7 +1,12 @@
-# local usage:
-#
-#   fly set-pipeline -t lite -n -c pipelines/zap.yml -p zap --load-vars-from config/local.yml -v script-branch=`git rev-parse --abbrev-ref HEAD`
-#
+<%# note that this pipeline file is actually an ERB template, and thus shouldn't be used with `fly` directly -%>
```

Comment:
ha, 🎩 ✨  this is my favorite.  Now you see me...
& making it obvious (`<%`) that ERB lies ahead.